config/locales/api/am_ET.yml

```diff
@@ -1,4 +1,4 @@
-am-ET:
+am_ET:
```

Comment:
IIRC, this key should not change. It should remain hyphenated. Let's revert this diff. 

Transifex annoyingly makes this diff every time ([documented here](https://github.com/simpledotorg/simple-server/blob/master/doc/wiki/transifex-localization.md#3-check-and-fix-all-top-level-language-keys))